### ISSUE TITLE
update environment.yml to Python 3.9

### ISSUE THREAD

**Issue by raybellwaves** [anitagraser/movingpandas#150]
Am I right in thinking the purpose of https://github.com/anitagraser/movingpandas/blob/master/environment.yml is for development?

I think it's ok to have one file unlike multiple environment files in geopandas (https://github.com/geopandas/geopandas/tree/master/ci/envs).

I would suggest moving python to 3.9 as the latest stable version. You have >=3.7 in the setup so it shouldn't cause any issues.

I also just realized it's missing pytest. You could also pin versions such as shapely https://github.com/anitagraser/movingpandas/blob/master/setup.py#L7 and add a comment related to an issue on why it's pinned.

**Comment by anitagraser** [comment]
Interestingly, Geopandas https://github.com/geopandas/geopandas/blob/master/environment.yml  doesn't even specify the Python version.

**Comment by anitagraser** [comment]
Might be also worthwhile to add different python versions to Travis conf (.travis.yml)

**Comment by raybellwaves** [comment]
> Interestingly, Geopandas https://github.com/geopandas/geopandas/blob/master/environment.yml doesn't even specify the Python version.

Good spot. I tested the geopandas env and you get latest python (3.9.7). My only hesitancy is when 3.10 is used there may be larger issues with upstream packages not being available etc. I'm also not sure what this file is for. Looking at the commit history is when when you setup MyBinder. Interesting they point you to a manual install of deps https://docs.geopandas.org/en/latest/community/contributing.html#installing-dependencies

> Might be also worthwhile to add different python versions to Travis conf (.travis.yml)

We can move this to a separate issue. I also made a note about GitHubActions in https://github.com/anitagraser/movingpandas/issues/143#issuecomment-956512789
